lib/src/anchor.dart

```diff
@@ -21,4 +21,27 @@ class Anchor {
   Vector2 translate(Vector2 p, Vector2 size) {
     return p - (toVector2..multiply(size));
   }
+
+  static final Map<Anchor, String> _valueNames = {
+    topLeft: 'topLeft',
+    topCenter: 'topCenter',
+    topRight: 'topRight',
+    centerLeft: 'centerLeft',
+    center: 'center',
+    centerRight: 'centerRight',
+    bottomLeft: 'bottomLeft',
+    bottomCenter: 'bottomCenter',
+    bottomRight: 'bottomRight',
+  };
+
+  static final List<Anchor> values = _valueNames.keys.toList();
+
+  @override
+  String toString() {
+    return _valueNames[this];
+  }
+
+  static Anchor parse(String name) {
```

Comment:
I think we should remove this method and the `_valueNames`, but keep the `values` one but just have a list of the predefined anchors in there, instead of taking the keys from the map.
If you don't have any good argument for why it is needed that is? Because otherwise I think it would lead to an anti-pattern where Anchors are created from strings, and that is not the intention here right?